CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Added a new validation (`PA137`) to **validate**, ensuring the absence of non-ig
 * Updated the **format** command to use the content graph instead of the id_set file.
 * Updated **format** command not to fail on unexpected values that returns from the graph, and just add it to the log.
 * Removed a redundant debug log on the `tools.get_file` function.
+* Adding a new boolean flag *auto-replace-uuids* to **download** command.
```

Comment:
After changing the flag name, make sure to add a description about the flag here:
Use this flag to avoid UUID replacements when downloading (which content items it affects) using **download** command.

Reply:
```suggestion
* Added the *auto-replace-uuids* flag to the **download** command. Use it to ... <what Yuval suggested>
```